Resources/Prototypes/_SpaceCats/Entities/Mobs/NPCs/slimes.yml

```
@@ -0,0 +1,198 @@
+  - type: XenoBiology
+    mutagen: MobAdultSlimesDarkBlue
+
+# Angry slimes
+
+- type: entity
+  name: grey slime
+  parent: MobAdultSlimesGrey
+  id: MobAdultSlimesGreyAngry
+  suffix: Angry
+  components:
+    - type: NpcFactionMember
```

yamllint (1.35.1)

[warning] wrong indentation: expected 2 but found 4

(indentation)